stl/debugger/STL.natvis

```diff
@@ -418,6 +425,14 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
       </Expand>
   </Type>
 
+  <Type Name="std::_Ref_count_obj_alloc3&lt;*&gt;">
+      <DisplayString>allocate_shared</DisplayString>
+      <Expand>
+          <Item Condition="_Uses != 0" Name="[original ptr]">($T1 *) &amp;_Storage</Item>
+          <Item Name="[allocator]">*((_Mybase *) this)</Item>

```

Comment:
This uses: https://github.com/microsoft/STL/blob/7f04137880e1c3df5125c1baf808f16f399dee2e/stl/inc/memory#L2399-L2403

However, I don't believe that this will work for stateful allocators: https://github.com/microsoft/STL/blob/7f04137880e1c3df5125c1baf808f16f399dee2e/stl/inc/memory#L2419-L2436